packages/n-back/docs/n-back.md

```diff
@@ -0,0 +1,51 @@
+# n-back
+
+Creating template for n-back experiments
+
+## Parameters
+
+### Initialization Parameters
+
+Initialization parameters can be set when calling `initJsPsych()`
+
+```js
+initJsPsych({
+  timelines: [
+    {type: jsPsychTimelineNBack, params: {...}}
+  ]
+})
+```
+
+Parameter | Type | Default Value | Description
+----------|------|---------------|------------
+          |      |               |
```

Comment:
probably best to either move the chart into here, or have both places have the parameters charts